bindings/experimental/distrdf/python/DistRDF/Backends/Dask/Backend.py

```diff
@@ -27,9 +27,13 @@
 class DaskBackend(Base.BaseBackend):
     """Dask backend for distributed RDataFrame."""
 
-    def __init__(self, daskclient=None):
+    def __init__(self, daskclient):
```

Comment:
Why is this not `None` by default anymore?

Reply:
I thought it was a bit redundant given how we construct the DaskBackend object in the `__init__.py` file of the module
```python
    from DistRDF.Backends.Dask import Backend
    daskclient = kwargs.get("daskclient", None)
    daskbackend = Backend.DaskBackend(daskclient=daskclient)
```
In practice, it should not matter. I can put it back to None again as default

Reply:
Ok, I just didn't see that!

Reply:
No problem! In fact, the Spark backend also has this as optional argument. Since it doesn't change much, I prefer the consistency. Maybe at some point we'll change both :+1: